src/Microsoft.AspNetCore.SignalR.Client.TS/Connection.ts

```diff
@@ -26,12 +28,12 @@ export class Connection {
         this.connectionState = ConnectionState.Disconnected;
     }
 
-    async start(transportName: string = 'webSockets'): Promise<void> {
+    async start(transportName: string = TransportType.webSockets): Promise<void> {
         if (this.connectionState != ConnectionState.Disconnected) {
             throw new Error("Cannot start a connection that is not in the 'Disconnected' state");
         }
 
-        this.transport = this.createTransport(transportName);
+        this.transport = new TransportFactory(this.httpClient).create(transportName);        
```

Comment:
Since httpClient is created considering the ISignalROptions in Connection class, I've changed it to create the instance in the Ctor.